BlueprintUICommonControls/Sources/Label.swift

```diff
@@ -33,15 +34,7 @@ public struct Label: Element {
             ])
     }
 
-    public var content: ElementContent {
-        var element = AttributedLabel(attributedText: attributedText)
-        element.numberOfLines = numberOfLines
-        element.roundingScale = roundingScale
-        return ElementContent(child: element)
-    }
-
-    public func backingViewDescription(bounds: CGRect, subtreeExtent: CGRect?) -> ViewDescription? {
-        return nil
+    public var elementRepresentation: Element {
+        AttributedLabel(attributedText: attributedText)
```

Comment:
Still need to set number of lines and rounding scale, no?

Reply:
Doh! Righto